Specs/Scene/SceneTransformsSpec.js

```diff
@@ -192,4 +192,19 @@ defineSuite([
         expect(drawingBufferCoordinates.x).toBeLessThan(1.0);
         expect(drawingBufferCoordinates.y).toBeLessThan(1.0);
     });
+
+    it('should not error when zoomed out and in 2D', function(done) {
+        var scene = createScene();
```

Comment:
Creating a `Scene` for this test is fine, but you need to call `destroyForSpecs` afterwards to release all WebGL resources.